index.rst

```diff
@@ -6,24 +6,499 @@
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+Portal tutorial format and style
+================================
+
+The portal tutorials are written in reStructuredText (RST) format and are kept within the data release documentation at
+`Portal Tutorials under DP0.2 Tutorials <https://dp0-2.lsst.io/tutorials-examples/index.html#portal-tutorials>`_.
+
+All portal tutorials should have a descriptive title, list the contact authors, the date last verified to run, and the targeted learning level.
+A brief narrative introduction to the tutorial should be provided at the top of the page.
+
+The rest of the portal tutorial should be divided into sequentially numbered steps and substeps.
+Use descriptive text and screenshots to demonstrate what the user should do.
+Augment screenshots with indicators (e.g., arrows or circles) to guide the users attention as needed.
+
+Ensure that any Astronomical Data Query Language (ADQL) is put into code boxes in RST so that users may copy-paste whenever possible.
+
+It is very common, but not mandatory, to end all portal tutorials with a section called ``Exercises for the learner`` with suggestions of
+how the user can make changes to the tutorial test options and examples, or guide them on the next step forward on their own.
+
+
+Git branch, merge, and review policy for portal tutorials
+=========================================================
+
+The following applies when creating or updating tutorials in the `dp0-2_lsst_io repository <https://github.com/lsst/dp0-2_lsst_io>`_, which is part of the ``lsst`` GitHub Organization.
+
+Develop new tutorials, or update existing ones, in a new branch.
+This branch should be named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+The new branch should be created from ``main``.
+Typically, only one tutorial is updated per ticket branch.
+
+Make commits and push changes to your branch in the ``dp0-2_lsst_io`` repository until work is complete, then open a pull request to ``main``.
+
+Contact one or more Rubin Observatory staff members with the appropriate expertise and ask them to review the notebook.
+At least one reviewer should be a member of the CET.
+If they agree, assign them as a reviewer on your pull request.
+
+Ensure that all of the reviewers' comments are addressed.
+Make changes and new commits to the branch, and respond to all of their comments with either a confirmation a change was made,
+or an explanation of why the request was not implemented.
+
+Contact the reviewers to let them know the pull request now awaits their approval.
+
+After the reviewers have approved the pull request, ``rebase and merge`` your ticket branch into the ``main`` branch.
+Resolve all conflicts, if there are any.
+After the successful merge, delete your branch.
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+Stretch goals
```

Comment:
MLG yes this is informal at this time; yes the point is to inspire people to work towards these stretch goals; no we do not have a specific timeline or tickets for this yet. I can see that a “NOTE” has been added to alert readers to the fact that these are stretch goals so that’s nice and clear.